### PR TITLE
feat(storage.support): add support component to a database general info view

### PR DESCRIPTION
**Pull request by brofik**
| Question         | Answer
| ---------------- | ---
| Branch?          | `release/sprint-2240-2242`  
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | yes
| Tickets          | Fix # MANAGER-9816 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Related
- [x] TRANS-46740

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/instances/instance/instance.component.js [hunk 1]
```diff
@@ -34,6 +34,8 @@ export default {
     shelveInstance: '<',
     unshelveInstance: '<',
     goToBlockStorages: '<',
+    goToSupportPage: '<',
+
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/instances/instance/instance.component.js:38]: **nitpick(non-blocking):** Could you please remove this empty line?
```suggestion
```

File: packages/manager/modules/pci/src/projects/project/instances/instance/instance.controller.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ export default class PciInstanceController {
     coreConfig,
     CucCloudMessage,
     ovhManagerRegionService,
+    OvhApiSupport,
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/instances/instance/instance.controller.js:8]: **nitpick:** Remove this unused dependency injection

**Review by anooparveti — CHANGES_REQUESTED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/instances/instance/instance.controller.js [hunk 3]
```diff
@@ -16,6 +16,9 @@ export default class PciInstanceController {
 
   $onInit() {
     this.loadMessages();
+    this.PciProjectsProjectInstanceService.getSupportLevel().then((value) => {
+      this.supportLevel = `pci_projects_project_instances_instance_support_level_${value.level}`;
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/instances/instance/instance.controller.js:20]: **question:** Isn't this supposed to be translation?
```suggestion
      this.supportLevel = this.$translate.instant(`pci_projects_project_instances_instance_support_level_${value.level}`);
```

Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/instances/instance/instance.controller.js:20]: It is. Thank your for noticing.

**Review by anooparveti — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/databases/database.service.js [hunk 4]
```diff
@@ -1118,4 +1120,8 @@ export default class DatabaseService {
       )
       .then(({ data }) => data);
   }
+
+  getSupportLevel() {
+    return this.OvhApiMe.v6().supportLevel().$promise;
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/storages/databases/database.service.js:1125]: **issue(blocking):** There's no need to perform an API call to retrieve this information. We already have it via `coreConfig.getUser()`.

**Review by antleblanc — CHANGES_REQUESTED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

File: packages/manager/modules/support/src/tickets/new-ticket/new-ticket.html [hunk 5]
```diff
@@ -20,6 +20,7 @@
 >
 </support-new-creation-form>
 
+<!-- comment to be deleted after QA validation -->
```
Comment (JeremyDec) [inline packages/manager/modules/support/src/tickets/new-ticket/new-ticket.html:23]: Could be deleted now no?
```suggestion
```

**Review by JeremyDec — APPROVED** [review]

**Pull request merged**